main/remoteservices/src/EBox/RemoteServices/RESTClient.pm

```diff
@@ -259,7 +260,19 @@ sub request {
     $ua->agent("ZentyalServer $version");
     $ua->ssl_opts('verify_hostname' => EBox::Config::boolean('rs_verify_servers'));
     # Set HTTP proxy if it is globally set as environment variable
-    $ua->proxy('https', $ENV{HTTP_PROXY}) if (exists $ENV{HTTP_PROXY});
+
+    my $proxy =  $ENV{HTTP_PROXY};
+    if (not $proxy) {
+        # check that if we have proxy set in network RO
+        my $network = EBox::Global->getInstance(1)->modInstance('network');
+        $proxy = $network->proxySettings(1);
```

Comment:
This requires a change in the requirements of zentyal-network minimum package to run.